frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.html

```diff
@@ -120,8 +120,18 @@
     ></span>
     <span
       class="ft-14-600 validation-message"
-      *ngIf="conditionCodeError"
-      [innerHTML]="conditionCodeError"
+      *ngIf="conditionUniqueCodeError"
```

Comment:
@ppratikcr7 Can you generalised this 3 set of span which is being used for displaying various conditions error? Instead of declaring 3 different variables to handle condition error, just use a single variable and handle the all condition errors would be great.

Reply:
Done, updated the PR.

<img width="1279" alt="Screenshot 2021-10-05 at 1 54 52 PM" src="https://user-images.githubusercontent.com/33730817/135988938-5476d8fc-315f-4364-8956-1e70ee327628.png">